src/components/SelectionList/Search/ReportInfoCell.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Avatar from '@components/Avatar';
+import Text from '@components/Text';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {getPolicy} from '@libs/PolicyUtils';
+import {getIcons, getReportName, getReportOrDraftReport} from '@libs/ReportUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+type ReportInfoCellProps = {
+    reportID: string;
+};
+
+function ReportInfoCell({reportID}: ReportInfoCellProps) {
```

Comment:
I think we should make this name more generic, maybe `AvatarWithTextCell` to improve reusability